README.md

```diff
@@ -1,9 +1,24 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Configuration
 
+1. Install dependencies.
 
+```
+npm i
+```
 
---- 
+2. Set up .env files for environment variables.
 
-This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+   There are two .env files required: .env.test and .env .development. These
+   should go in root directory
+
+```
+echo "PGDATABASE=database_name_here" > .env.test
+echo "PGDATABASE=database_name_here" > .env.development
```

Comment:
Really nice way of creating these two files, but what should the database names be? Or where can these be found?